src/stores/GlobalStore.ts

```diff
@@ -48,6 +53,7 @@ export const useGlobalStore = defineStore('globalStore', {
                     },
                 },
             ],
+            userStarAtlasNfts: [] as UserNft[],
```

Comment:
why do we store the userNft in gloabl store?

Reply:
Do you think it would make more sense to create dedicated store for `userStarAtlasNfts` and `userTokens`?

Reply:
for tokens there is an object which can be used for filtering https://github.com/skullnbonesdao/trade.skullnbones.xyz/blob/master/src/typescript/constants/tokens.ts and the current selected is stored inside gloablStore() there is also a functions to change the asset by name

Reply:
userStarAtlasNfts what do we store in there?